### DISCUSSION TITLE
How to make the frontend live server run inside Docker?

### DISCUSSION THREAD

**Discussion by poyang1024** [fastapi/full-stack-fastapi-template#1181] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched in the documentation/README.
- [X] I already searched in Google "How to do X" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs/README and didn't find an answer.

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

I try to rewrite the files like Dockerfile, .env to make this repo's frontend development env run inside the docker. However, it is still not working, I need more help. Thank you.

### Description

I want to know how to make this repo's frontend development env run inside the docker, it can let me more productive to running the local development server with live reload.

### Operating System

macOS

### Operating System Details

use mac to run docker-compose

### Python Version

python3.10

**Comment by YuriiMotov** [comment, marked as answer]
It's advised in docs that you run frontend development server locally (not in the container): https://github.com/fastapi/full-stack-fastapi-template/blob/master/development.md#local-development

Here are some instructions on how to setup environment: https://github.com/fastapi/full-stack-fastapi-template/blob/master/frontend/README.md#frontend-development
